### PR TITLE
Adds Project.getDependentsGraph

### PR DESCRIPTION
**Pull request by lukebatchelor**
And some tests for the rest of Project

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,19 +8,16 @@
   "bin": {
```
Comment (lukebatchelor) [inline package.json]: This was before I removed json from prettiers list, happy to revert

File: src/__tests__/Project.test.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    it("should be able to getDependencyGraph", async () => {
+      const workspaces = await project.getWorkspaces();
+      const deps = await project.getDependencyGraph(workspaces);
+      const { valid, graph } = deps;
+
+      expect(valid).toEqual(true);
+      expect(graph).toBeInstanceOf(Map);
+
+      /**
+       Need to figure out if this workspace package should be included here or not, currently it is.
```
Comment (lukebatchelor) [inline src/__tests__/Project.test.js]: I can probably remove this now?

File: package.json (hunk 1 above)
Comment (thejameskyle) [reply package.json]: What?

**Comment by thejameskyle** [comment]
Can you turn on single quotes in prettier? That's whats currently being used and I'd rather not rewrite them all unnecessarily

**Comment by lukebatchelor** [comment]
Addressed comments 👍

**Pull request merged**